Makefile

```diff
@@ -21,18 +21,29 @@ help : Makefile
 	@sed -n 's/^##//p' $<
 
 ## build                   : PHONY, builds distribution
-## upload                  : PHONY, uploads to internal pypiserver
-## test                    : PHONY, runs pytest
 build: tmp/.sentinel.build
+## upload                  : PHONY, uploads to internal pypiserver
 upload: tmp/.sentinel.upload
+## test                    : PHONY, runs pytest
 test: tmp/.sentinel.test
+## discover                : PHONY, runs the tap in discovery mode
+discover: tmp/.sentinel.venv-dev
+	@echo '{'                                 >  fake_config.json
+	echo  '  "token"     : "fake_token",'     >> fake_config.json
```

Comment:
Do we use a fake token here because we don't actually contact the API to do discovery for this tap?